README.md

```diff
@@ -198,7 +158,6 @@ jobs:
 ## Notes
 
 - A GitHub token is automatically provided by Github Actions, which can be accessed using `github.token` and supplied to the action as an input `github-token`.
```

Comment:
this part could be reworded because now the action automatically uses the built-in token (it's the default value). we can keep this sentence, but instead we should specify that the github-token input can be used if you want to provide a token that's not the default one